src/cachers/collections/organisations.js

```diff
@@ -1,18 +1,49 @@
 import baseData from './index.js';
 import organizationsMixin from '../../mixins/europeana/entities/organizations.js';
+import { langMapValueForLocale } from '../../plugins/europeana/utils.js';
+import { createEuropeanaApiClient } from '../utils.js';
 
-const PICK = ['slug', 'prefLabel'];
+const PICK = ['slug', 'recordCount', 'nativeLabel', 'nonNativeEnglishLabel'];
 const INTERNATIONALISE = (entities) => entities.map((entity) => ({
   ...entity,
-  prefLabel: organizationsMixin.methods.organizationEntityNativeName(entity)
+  nativeLabel: langMapValueForLocale(organizationsMixin.methods.organizationEntityNativeName(entity)),
+  nonNativeEnglishLabel: langMapValueForLocale(organizationsMixin.methods.organizationEntityNonNativeEnglishName(entity))
 }));
-const SORT = (entity) => Object.values(entity.prefLabel)[0];
 
-const data = (config = {}) => baseData({ type: 'organization' }, config);
+let axiosClient;
+
+async function getRecordCounts() {
+  const params = {
+    profile: 'facets',
+    query: 'foaf_organization:*data.europeana.eu*',
+    facet: 'foaf_organization',
+    ['f.foaf_organization.facet.limit']: 10000,
+    rows: 0
+  };
+  const response = await axiosClient.get('/search.json', { params });
+  return response.data?.facets?.[0]?.fields || [];
+}
+
+const data = async(config = {}) => {
+  const baseOrgData = await baseData({ type: 'organization' }, config);
+
+  axiosClient = createEuropeanaApiClient(config.europeana?.apis?.record);
+
+  const recordCounts = await getRecordCounts();
+
+  Object.keys(baseOrgData).forEach(
```

Comment:
baseOrgData should be an array, so Instead of this, just use:
```
for (const organisation of baseOrgData) {
```

Reply:
You're right, it's an array. I've used `.map` instead.